test/libsolidity/semanticTests/array/delete_storage_array.sol

```diff
@@ -0,0 +1,27 @@
+contract C {
+    uint[] data;
+
+    function len() public returns (uint ret) {
+        data.push(234);
+        data.push(123);
+        delete data;
+        assembly {
+            ret := sload(data_slot)
+        }
+    }
+
+    function val() public returns (uint ret) {
+        data.push(234);
+        data.push(123);
+        delete data;
+        assembly {
+            ret := sload(keccak256(data_slot, 32))
```

Comment:
```suggestion
            mstore(0, data_slot)
            ret := sload(keccak256(0, 32))
```

Reply:
Actually to check that the test does what it is supposed to do, it would be better to compute the slot, store something there via inline assembly and then retrieve it via solidity. Then run delete and check that the same slot is cleared now as you do here.